rebase-helper.spec

```diff
@@ -42,6 +42,8 @@ Recommends:     licensecheck
 Recommends:     rpmlint
 Recommends:     libabigail
 Recommends:     pkgdiff >= 1.6.3
+Recommends:     rpminspect
+Recommends:     rpminspect-data-fedora
```

Comment:
Is `rpminspect-data-fedora` really needed? As far as I can see, it is required by `librpminspect`, which is a dependency of `rpminspect`.

Reply:
Good catch, I followed the README of rpminspect but `rpminspect-data-fedora` doesn't seem to be necessary.